### PR TITLE
Fetch hello message from server.

### PR DESCRIPTION
**Pull request by bettyashley**
Fetch a 'Hello Betty!' message from the data server and adds it to the DOM.

demo = https://bettyv-step-2020.appspot.com/

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -34,3 +34,12 @@ function addRandomGreeting() {
...
     document.body.style.background = color;
 }
+
+/**
+ * Fetches 'Hello Betty!' message from the server and adds it to the DOM.
+ */
+async function getHelloMessage() {
+    const response = await fetch('/data');
+    const message = await response.text();
+    document.getElementById('message-container').innerText = message;
```
Comment (Enriqe) [inline portfolio/src/main/webapp/script.js:44]: `textContent` is preferred over `innerText`, it's faster and potentially safer

https://dev.to/4myc/javascript-innerhtml-innertext-and-textcontent-ih

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -34,3 +34,12 @@ function addRandomGreeting() {
 function changeBackgroundColor(color) {
     document.body.style.background = color;
 }
+
+/**
+ * Fetches 'Hello Betty!' message from the server and adds it to the DOM.
+ */
+async function getHelloMessage() {
```
Comment (Enriqe) [inline portfolio/src/main/webapp/script.js:41]: Let's make this function name a bit more generic, same for the the comment. This helps us prevent having to change the name of the function / comment on every change.

I'm thinking: `getGreetingMessage()` and a more generic comment above.

**Comment by Enriqe** [comment]
Could you push the changes to the demo please?

**Comment by bettyashley** [comment]
Demo has been updated!

**Comment by Enriqe** [comment]
Can you try to wrap the text to fit the screen? It's bleeding out in mobile

image


(To test in mobile view, go to Chrome "view" menu > Developer > Developer Tools

**Comment by Enriqe** [comment]
Can you please push the changes to the demo site so I can verify the fix?

**Comment by bettyashley** [comment]
It should now work on the demo for mobile, but it now looks different on a desktop-screen let me know if you want that changed back.

**Pull request merged**